src/main/scala/org/coursera/autoschema/AutoSchema.scala

```diff
@@ -16,9 +16,7 @@
 
 package org.coursera.autoschema
 
-import play.api.libs.json.JsArray
-import play.api.libs.json.Json
-import play.api.libs.json.JsObject
+import play.api.libs.json.{JsString, JsArray, Json, JsObject}
```

Comment:
Coursera's Scala style guide is to put these each on separate lines. (This makes PR's cleaner when adding and removing imports.) Could you make this change?